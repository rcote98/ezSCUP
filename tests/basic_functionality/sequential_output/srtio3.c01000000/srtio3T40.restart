2	2	2
5	3
Sr	Ti	O
3.00000000E-02	3.00000000E-02	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	-8.23875882E-02	2.18468526E-02	1.31722957E-03
0	0	0	2	2	-9.76914899E-02	7.68368204E-02	-2.27371099E-02
0	0	0	3	3	2.41539101E-01	5.27485182E-02	-5.50312240E-03
0	0	0	4	3	3.82493284E-02	-3.89825894E-02	-1.10308006E-01
0	0	0	5	3	4.63829496E-02	2.29614951E-03	-1.34707171E-01
0	0	1	1	1	-4.53959287E-02	5.47881424E-02	2.82715648E-03
0	0	1	2	2	-9.85936026E-02	6.68479300E-02	-2.27677596E-02
0	0	1	3	3	-6.14890556E-02	-2.07655443E-01	3.78325627E-03
0	0	1	4	3	-5.53835304E-03	-4.12529362E-02	1.15505252E-01
0	0	1	5	3	4.52811666E-02	-2.56287633E-04	1.30341667E-01
0	1	0	1	1	-2.97290743E-02	3.76915313E-02	-1.69851751E-02
0	1	0	2	2	-9.86411498E-02	8.36549755E-02	-4.84787959E-03
0	1	0	3	3	-8.48169038E-02	-1.94472745E-01	-4.33467242E-03
0	1	0	4	3	3.72166912E-02	-4.72811407E-02	1.21099039E-01
0	1	0	5	3	5.59467959E-02	-1.30046323E-02	1.59078416E-01
0	1	1	1	1	-4.82809902E-02	5.15557052E-02	8.00866545E-03
0	1	1	2	2	-1.02619558E-01	7.94377284E-02	-5.75319367E-04
0	1	1	3	3	2.55411468E-01	4.19774293E-02	-5.47745766E-03
0	1	1	4	3	2.70260005E-02	-3.42359069E-02	-1.06524577E-01
0	1	1	5	3	4.38942990E-02	-2.94130733E-02	-1.57733636E-01
1	0	0	1	1	-4.68612471E-02	4.94031744E-02	-2.23382480E-03
1	0	0	2	2	-9.50191994E-02	1.09948280E-01	1.58703758E-03
1	0	0	3	3	-3.77851382E-02	-2.42193416E-01	-1.04369632E-03
1	0	0	4	3	4.22587729E-02	-4.44925808E-02	1.88479850E-01
1	0	0	5	3	4.82337069E-02	-6.10949333E-03	1.27292878E-01
1	0	1	1	1	-5.51996122E-02	3.53479436E-02	1.02439085E-02
1	0	1	2	2	-1.06734491E-01	1.14592597E-01	8.06126161E-03
1	0	1	3	3	2.14821812E-01	8.95077168E-02	1.39803833E-02
1	0	1	4	3	1.23724530E-02	-5.14096289E-02	-1.33620037E-01
1	0	1	5	3	4.07171566E-02	-3.10745000E-02	-1.23622378E-01
1	1	0	1	1	-8.30951594E-02	2.75822330E-02	1.74786565E-02
1	1	0	2	2	-1.03751420E-01	8.77610798E-02	-4.88012894E-03
1	1	0	3	3	2.63014446E-01	8.27416561E-02	-1.32600814E-02
1	1	0	4	3	1.44327121E-02	-4.71076110E-02	-1.63835503E-01
1	1	0	5	3	5.55798542E-02	-1.08466356E-02	-1.74098971E-01
1	1	1	1	1	-5.00656899E-02	6.52512921E-02	-1.08154008E-02
1	1	1	2	2	-1.05410738E-01	1.03031280E-01	-4.19757201E-03
1	1	1	3	3	-7.03288452E-02	-2.69325704E-01	-6.94329726E-03
1	1	1	4	3	-1.71800980E-02	-4.74812438E-02	1.77013719E-01
1	1	1	5	3	4.42366192E-02	2.17465378E-02	1.44954398E-01
