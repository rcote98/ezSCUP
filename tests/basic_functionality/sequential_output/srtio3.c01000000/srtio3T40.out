
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T40.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298474E+05
Polarization (C/m2):    -0.192801    0.168910   -0.006252
Strain:                  0.030000    0.030000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.400000E+02 -0.298473E+05 -0.131942E+00  0.209243E+00 -0.283041E-01 -0.193611E+00  0.206658E+00 -0.119086E-01  0.300000E-01  0.300000E-01 -0.358225E-02 -0.128615E-02 -0.132094E-02  0.000000E+00  0.330000E-01  0.330000E-01 -0.205975E-02 -0.479198E-03 -0.100230E-02  0.000000E+00 
LT:      20  0.400000E+02 -0.298473E+05 -0.180340E+00  0.212296E+00 -0.491119E-01 -0.181802E+00  0.201416E+00 -0.282959E-01  0.300000E-01  0.300000E-01 -0.548606E-02  0.983437E-03 -0.158144E-02  0.000000E+00  0.315000E-01  0.315000E-01 -0.333251E-02 -0.264078E-03 -0.129092E-02  0.000000E+00 
LT:      30  0.400000E+02 -0.298473E+05 -0.222084E+00  0.223393E+00  0.590076E-02 -0.193014E+00  0.205068E+00 -0.247982E-01  0.300000E-01  0.300000E-01 -0.483834E-02  0.119171E-02 -0.243473E-02  0.000000E+00  0.310000E-01  0.310000E-01 -0.393645E-02  0.150911E-03 -0.154905E-02  0.000000E+00 
LT:      40  0.400000E+02 -0.298473E+05 -0.167153E+00  0.249266E+00 -0.158963E-01 -0.196216E+00  0.211514E+00 -0.189675E-01  0.300000E-01  0.300000E-01 -0.865131E-02  0.980691E-03 -0.600090E-03  0.000000E+00  0.307500E-01  0.307500E-01 -0.460046E-02  0.293226E-03 -0.145797E-02  0.000000E+00 
LT:      50  0.400000E+02 -0.298474E+05 -0.205420E+00  0.206262E+00 -0.230747E-01 -0.195200E+00  0.212544E+00 -0.228612E-01  0.300000E-01  0.300000E-01 -0.891105E-02  0.132529E-02 -0.346088E-02  0.000000E+00  0.306000E-01  0.306000E-01 -0.563018E-02  0.269986E-03 -0.134151E-02  0.000000E+00 
LT:      60  0.400000E+02 -0.298474E+05 -0.188509E+00  0.261793E+00  0.118460E-01 -0.198157E+00  0.213583E+00 -0.209476E-01  0.300000E-01  0.300000E-01 -0.103446E-01  0.224558E-02 -0.360168E-02  0.000000E+00  0.305000E-01  0.305000E-01 -0.627424E-02  0.340734E-03 -0.175024E-02  0.000000E+00 
LT:      70  0.400000E+02 -0.298474E+05 -0.173503E+00  0.241828E+00  0.425239E-01 -0.195614E+00  0.216124E+00 -0.146004E-01  0.300000E-01  0.300000E-01 -0.144980E-01  0.107605E-02 -0.423185E-03  0.000000E+00  0.304286E-01  0.304286E-01 -0.717000E-02  0.466077E-03 -0.182956E-02  0.000000E+00 
LT:      80  0.400000E+02 -0.298475E+05 -0.211308E+00  0.212409E+00  0.211454E-01 -0.198255E+00  0.214779E+00 -0.922482E-02  0.300000E-01  0.300000E-01 -0.172245E-01  0.842835E-03  0.605980E-03  0.000000E+00  0.303750E-01  0.303750E-01 -0.839912E-02  0.560661E-03 -0.129289E-02  0.000000E+00 
LT:      90  0.400000E+02 -0.298475E+05 -0.202769E+00  0.206507E+00  0.273747E-01 -0.199412E+00  0.214524E+00 -0.544149E-02  0.300000E-01  0.300000E-01 -0.171165E-01  0.421438E-02 -0.468628E-02  0.000000E+00  0.303333E-01  0.303333E-01 -0.941863E-02  0.807749E-03 -0.154294E-02  0.000000E+00 
LT:     100  0.400000E+02 -0.298475E+05 -0.208497E+00  0.184911E+00  0.149727E-01 -0.201799E+00  0.213142E+00 -0.252020E-02  0.300000E-01  0.300000E-01 -0.190427E-01  0.270589E-02 -0.431572E-02  0.000000E+00  0.303000E-01  0.303000E-01 -0.102627E-01  0.110238E-02 -0.189320E-02  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298475E+05
Polarization (C/m2):    -0.208497    0.184911    0.014973
Strain:                  0.030000    0.030000   -0.019043    0.002706   -0.004316    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T40
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T40.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          40.00000000         kelvin
# above item originally: MC_temperature          40.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.78       3.78       3.78     0
  setup_readinput                3.56       3.56       3.55     0
  montecarlo_main                0.21       0.21       0.21     0
  montecarlo_sweep               0.21       0.21       0.21     0
  io_readparameterfile           3.56       3.56       3.55     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.993 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
